docs/quick-start/cassandra-cluster.yaml

```diff
@@ -31,3 +31,5 @@ spec:
   pilotImage:
     repository: "quay.io/jetstack/navigator-pilot-cassandra"
     tag: "v0.1.0"
+  securityContext:
+    runAsUser: 999
```

Comment:
As per discussion earlier - what happens if this field is not specified on a Cassandra resource?

If we are not careful, this PR could become a breaking change as older versions of Navigator handled setting the uid automatically (i.e. via the entrypoint script) before forking to the cassandra subprocess.